pkg/tenant/db/tenants_test.go

```diff
@@ -59,58 +59,65 @@ func (s *dbTestSuite) TestCreateTenant() {
 	require.NoError(err, "could not create tenant")
 
 	// Fields should have been populated
-	require.NotEqual(uuid.Nil, tenant.ID, "expected non-zero uuid to be populated")
+	require.NotEqual("", tenant.ID, "expected non-zero ulid to be populated")
```

Comment:
I suspect that `NotZero` should work with a ulid -- but I'm not 100% sure. 

```suggestion
	require.NotZero(tenant.ID, "expected non-zero ulid to be populated")
```

Reply:
Yes, it'll work. I'll have to add an ID to that test.